src/frontend/js/utils/AbilitiesHelper/joanieUserProfileAbilities.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { JoanieUserApiAbilityActions } from 'types/User';
+import { JoanieUserProfileFactory } from 'utils/test/factories/joanie';
+import { JoanieUserProfileActions } from './types';
+import AbilitiesHelper from '.';
+
+describe('Abilities.JoanieUserProfile', () => {
+  it.each([
+    {
+      label: 'User has access to both a course and an organization',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: true,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: true,
+      },
+    },
+    {
+      label: 'User has access to a course',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: false,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: true,
+      },
+    },
+    {
+      label: 'User has access to an organization',
+      abilities: {
+        [JoanieUserApiAbilityActions.HAS_ORGANIZATION_ACCESS]: true,
+        [JoanieUserApiAbilityActions.HAS_COURSE_ACCESS]: false,
+      },
+    },
+  ])('should allow ACCESS_TEACHER_DASHBOARD when "$label"', ({ abilities }) => {
+    const joanieUser = JoanieUserProfileFactory({
+      abilities,
+    }).one();
+    expect(
+      AbilitiesHelper.can(joanieUser, JoanieUserProfileActions.CAN_ACCESS_TEACHER_DASHBOARD),
+    ).toBeTruthy();
+    expect(
+      AbilitiesHelper.cannot(joanieUser, JoanieUserProfileActions.CAN_ACCESS_TEACHER_DASHBOARD),
+    ).toBeFalsy();
+  });
+
+  it("shouldn't allow ACCESS_TEACHER_DASHBOARD when user isn't has access to neither course or organization", () => {
```

Comment:
Shouldn't allow access to the teacher dashboard when the user doesn't have access to either the course or the organization.*

Reply:
The typo is still here

```suggestion
  it("shouldn't allow access to the teacher dashboard when the user doesn't have access to either the course or the organization", () => {
```